src/vmm/src/devices/virtio/net/device.rs

```diff
@@ -870,6 +870,15 @@ impl VirtioDevice for Net {
     fn is_activated(&self) -> bool {
         self.device_state.is_activated()
     }
+
+    fn reset(&mut self) -> Result<(), ResetError> {
+        self.device_state = DeviceState::Inactive;
+        self.rx_bytes_read = 0;
+        self.rx_deferred_frame = false;
+        self.rx_frame_buf = [0u8; MAX_BUFFER_SIZE];
+        self.metrics = NetMetricsPerDevice::alloc(self.id.clone());
```

Comment:
We definitely need to reset the `acked_features` here, too.

Also, @sudanl0 do you think it is ok to reset the metrics here? I think it's more of a philosophical question, on whether we want our metrics to reflect the fact that the device has been reset by zeroing them out. Maybe, it would make sense to add a reset counter metric as well.